pkg/serverless/serverless.go

```diff
@@ -238,3 +239,14 @@ func computeTimeout(now time.Time, deadlineMs int64, safetyBuffer time.Duration)
 	currentTimeInMs := now.UnixNano() / int64(time.Millisecond)
 	return time.Duration((deadlineMs-currentTimeInMs)*int64(time.Millisecond) - int64(safetyBuffer))
 }
+
+func functionArnWithoutAlias(functionArn string) string {
```

Comment:
The last part of the function Arn is actually called a qualifier. It can be an alias but it can also be a specific version number.

Also, I would name functions as verbs, and the things they return as nouns.

So, I would rename this function `removeQualifierFromArn()`